server/game/cards/13-CW/ChancellorsAide.js

```diff
@@ -0,0 +1,44 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+const { Players, TargetModes } = require('../../Constants');
+
+class ChancellorsAide extends DrawCard {
+    setupCardAbilities() {
+        this.interrupt({
+            title: 'Player discards a card',
+            when: {
+                onCardLeavesPlay: (event, context) => event.card === context.source
+            },
+            targets: {
+                myPlayer: {
+                    mode: TargetModes.Select,
+                    targets: true,
+                    choices: {
+                        [this.owner.name]: AbilityDsl.actions.chosenDiscard(context => ({ target: context.player })),
```

Comment:
Should probably change this to "Me" & "My Opponent" instead of trying to get the names without a context object.  Not sure why it works with BHC but not here.